production/helm/loki/templates/index-gateway/statefulset-index-gateway.yaml

```diff
@@ -13,8 +13,7 @@ metadata:
 spec:
   replicas: {{ .Values.indexGateway.replicas }}
   updateStrategy:
-    rollingUpdate:
-      partition: 0
+    type: {{ .Values.indexGateway.updateStrategy }}
```

Comment:
this way it would be impossible for someone to set the former config (rollingUpdate: partition: 0).
can you please modify this to receive the whole updateStrategy?
Example:
```
{{- with .Values.indexGateway.updateStrategy }}
  updateStrategy:
     {{- tpl (. | toYaml) $ | nindent 4 }}
{{- end }}
```
And then set the value on `values.yaml` to be the old one (rollingUpdate: partition: 0)